vars.tf

```diff
@@ -68,3 +68,15 @@ variable "dd_forwarder_template_version" {
   type        = string
   default     = "3.13.0"
 }
+
+variable "excluded_regions" {
+  description = "An array of AWS regions to exclude from metrics collection"
+  type        = list(string)
+  default     = []
+}
+
+variable "filter_tags" {
+  description = "Array of EC2 tags (in the form key:value) defines a filter that Datadog use when collecting metrics from EC2. Only hosts that match one of the defined tags will be imported into Datadog. The rest will be ignored."
```

Comment:
Minor request. From the datadog provider documentation, there is mention of `Wildcards, such as ? (for single characters) and * (for multiple characters) can also be used. `, could you add that to the description as well?